### PR TITLE
Update trec_eval Fix #475

### LINKED ISSUE #475: Running trec_eval gives segmentation fault
The problem likely occurs because a old version of trec_eval is used. This [issue](https://github.com/usnistgov/trec_eval/issues/14) describes the problem. It will likely be fixed if the trec_eval in this repository is updated to the latest version.

### PR DESCRIPTION
**Pull request by Chriskamphuis**
Updated trec_eval to latest version. Running old version can result in a
segmentation fault.

### CODE REVIEW

**Comment by lintool** [comment]
Hi @Chriskamphuis I only see removal of the existing tarball? There isn't a new one?

**Comment by lintool** [comment]
Ah, I see - misread.

The new tarball is created from https://github.com/usnistgov/trec_eval/ ?
I think it would reduce confusion to change the name? I think the latest version is 9.0.4? Might want to include the GitHub commit id in this PR to be absolutely clear.

**Comment by Chriskamphuis** [comment]
@lintool 

I included the commit id in the commit that changed the name, the commit id of usnistgov/trec_eval is a5211566d0c9e2ec337bacf327b9350ab5b3edde.

**Comment by lintool** [comment]
@Chriskamphuis sorry to be kinda a pain, but we're try to implement rigorous software engineering practices in this project - CI is failing because of this -

https://github.com/castorini/Anserini/blob/master/.travis.yml#L16

File name needs to be updated. Can you please do so?

**Comment by Chriskamphuis** [comment]
@lintool Yes, no problem, I am working on it. I will also update the markdown files so it will be consistent.

File: docs/experiments-cw09b.md [hunk 1]
```diff
@@ -60,47 +60,47 @@ nohup target/appassembler/bin/SearchCollection -topicreader Webxml -index lucene
 Evaluation can be performed using `trec_eval` and `gdeval.pl`:
 
 ```
-eval/trec_eval.9.0/trec_eval -m map -m P.30 src/main/resources/topics-and-qrels/qrels.web.51-100.txt run.cw09b.bm25.topics.web.51-100.txt
```
Comment (Peilin-Yang) [inline docs/experiments-cw09b.md]: It is a bit odd why there is an extra empty line here.
Did you manually change this file?

Comment (lintool) [reply docs/experiments-cw09b.md]: @Peilin-Yang are those parts auto-gen'ed? Perhaps @Chriskamphuis doesn't know about the details of the regression framework...

Comment (Peilin-Yang) [reply docs/experiments-cw09b.md]: All `experiments-{$collection}.md` files are auto-generated. 
@Chriskamphuis Could you please run `mvn test` locally and see if there is any diff?

Comment (Chriskamphuis) [reply docs/experiments-cw09b.md]: @Peilin-Yang No diff for me locally. I only changed the the index.md and yaml files. Then I ran test before pushing the changes. Maybe I accidentally added an extra line when checking if everything was correct, but then it should be removed when running ``` mvn test ```  right?

**Comment by lintool** [comment]
@Chriskamphuis thanks for your first contribution. Looking forward to many more in the future!

**Pull request merged**